apps/store/extensions/app/social-reviews/modules/reviews-utils.js

```diff
@@ -61,17 +61,17 @@ var ReviewUtils = {};
         var myIndex;
         for (var index = 0; index < reviews.length; index++) {
             review = reviews[index];
-            review.user = false;
             usernameOnReview = review.actor.id;
             var targetId = String(review.object.id);
             review.actor.id = cleanUsername(review.actor.id);
             review.isMyComment = (usernameOnReview === formatUsername(user));
+            review.isUserLoggedIn = false;
             //Only populate review details if there is a logged in
             //user
             if (user && review.isMyComment) {
                 myIndex = index;
             } else if (user) {
-                review.user = true;
+                review.isUserLoggedIn = true
```

Comment:
Thanks, Make the changes accordingly  [commit](https://github.com/wso2/carbon-store/pull/733/commits/c85a36cab58dfb791d44bd0c9cabc17afb0db982)